### PR TITLE
fix: fix small ui issues after merge of multiple PRs

### PR DESCRIPTION
**Pull request by Bilb**

### CODE REVIEW

File: ts/components/conversation/message/message-item/InteractionNotification.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ export const InteractionNotification = (props: PropsForInteractionNotification)
 
   switch (interactionType) {
     case ConversationInteractionType.Hide:
-      text = window.i18n('hideConversationFailedPleaseTryAgain');
+      // this can't happen
```
Comment (yougotwill) [inline ts/components/conversation/message/message-item/InteractionNotification.tsx:41]: Please add a NOTE infront for easy searching later

File: ts/components/conversation/right-panel/overlay/message-info/components/AttachmentInfo.tsx [hunk 2]
```diff
@@ -10,7 +10,8 @@ type Props = {
 
 const StyledLabelContainer = styled(Flex)`
   div {
-    flex-grow: 1;
+    min-width: 50%;
+    // we want 2 items per row and that's the easiest to make it happen
```
Comment (yougotwill) [inline ts/components/conversation/right-panel/overlay/message-info/components/AttachmentInfo.tsx:14]: Please add NOTE infront for convenience

File: ts/components/leftpane/conversation-list-item/InteractionItem.tsx [hunk 3]
```diff
@@ -58,14 +58,7 @@ export const InteractionItem = (props: InteractionItemProps) => {
...
     case ConversationInteractionType.Hide:
-      errorText = window.i18n('hideConversationFailed');
-      text =
-        interactionStatus === ConversationInteractionStatus.Error
-          ? errorText
-          : interactionStatus === ConversationInteractionStatus.Start ||
-            interactionStatus === ConversationInteractionStatus.Loading
-          ? window.i18n('hiding')
-          : text;
+      // if it's hidden or pending hiding, we don't show it any text
```
Comment (yougotwill) [inline ts/components/leftpane/conversation-list-item/InteractionItem.tsx:61]: NOTE maybe? Or is it overkill?

File: ts/interactions/conversations/unsendingInteractions.ts [hunk 4]
```diff
@@ -398,6 +401,9 @@ export async function deleteMessagesById(messageIds: Array<string>, conversation
         window.inboxStore?.dispatch(resetRightOverlayMode());
       },
       closeAfterInput: false,
+      onClickClose: () => {
+        window.inboxStore?.dispatch(updateConfirmModal(null));
```
Comment (yougotwill) [inline ts/interactions/conversations/unsendingInteractions.ts:405]: closeDialog() ?

**Review by yougotwill — CHANGES_REQUESTED** [review]

File: ts/interactions/conversations/unsendingInteractions.ts (hunk 4 above)
Comment (Bilb) [reply ts/interactions/conversations/unsendingInteractions.ts:405]: closeDialog was not declared in that function so I've created it

File: ts/components/conversation/message/message-item/InteractionNotification.tsx (hunk 1 above)
Comment (Bilb) [reply ts/components/conversation/message/message-item/InteractionNotification.tsx:41]: I actually like it better as part of the case itself

File: ts/components/leftpane/conversation-list-item/InteractionItem.tsx (hunk 3 above)
Comment (Bilb) [reply ts/components/leftpane/conversation-list-item/InteractionItem.tsx:61]: there is a text, what do you mean?

Comment (yougotwill) [reply ts/components/leftpane/conversation-list-item/InteractionItem.tsx]: I meant doing something like `// NOTE this is a comment` it's how I have been doing comments over the last few months but I think it's overkill so don't worry about it.

**Review by yougotwill — APPROVED** [review]

**Pull request merged**